gc/base/standard/ConcurrentGC.cpp

```diff
@@ -2716,7 +2672,7 @@ MM_ConcurrentGC::doConcurrentTrace(MM_EnvironmentBase *env,
 
 	if (!isGcOccurred) {
 		/* If no more work left (and concurrent scanning is complete or disabled) then switch to exhausted now */
-		if (((MM_ConcurrentCardTable *)_cardTable)->isCardCleaningComplete() &&
+		if (NULL != _cardTable && ((MM_ConcurrentCardTable *)_cardTable)->isCardCleaningComplete() &&
```

Comment:
Would you please add extra pair of brackets around (NULL != _cardTable) - here and below ?

Reply:
fixed